python/pyarrow/tests/parquet/common.py

```diff
@@ -176,8 +176,8 @@ def alltypes_sample(size=10000, seed=0, categorical=False):
         # TODO(wesm): Test other timestamp resolutions now that arrow supports
         # them
         'datetime': np.arange("2016-01-01T00:00:00.001", size,
-                              dtype='datetime64[ms]').astype('datetime64[ns]'),
-        'timedelta': np.arange(0, size, dtype="timedelta64[ns]"),
+                              dtype='datetime64[ms]'),
```

Comment:
We can maybe keep both original ns and new ms resolution? (to test both)

Reply:
Good suggestion!